### PR TITLE
Release obj now works again for admins!

### PR DESCRIPTION
**Pull request by Ktlwjec**
Re-adds release obj as a right-click option for admins with +DEBUG (?). Fixes that god awful bug that blacks your screen by preventing release obj being used unless you're in possesion of an object.

### CODE REVIEW

**Comment by Moja10** [comment]
Oh my god I thought this was never going to get fixed and you did it!!!!!! very much thanks and appreciated!!!!!

File: code/modules/admin/verbs/possess.dm [hunk 1]
```diff
@@ -25,18 +25,24 @@
...
-		if(ishuman(usr))
-			var/mob/living/carbon/human/H = usr
-			H.name = H.get_visible_name()
+//YOGS start
+	if(!usr.control_object)
+		to_chat(usr, "You need to possess an object first!")
+		return
+//YOGS end
+
+		if(usr.control_object && usr.name_archive) //if you have a name archived and if you are actually relassing an object
```
Comment (Moja10) [inline code/modules/admin/verbs/possess.dm]: Be sure to modularize this section too!!

Comment (Amelia0010) [inline code/modules/admin/verbs/possess.dm]: This line is indented 2 twice, while the `if(!usr.control_object)` is only indented once, meaning this only triggers if they don't have have a control_object (even though there's a return in front of it)

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/possess.dm (hunk 1 above)
Comment (Ktlwjec) [reply code/modules/admin/verbs/possess.dm]: it is :thonkn:

Comment (Ktlwjec) [reply code/modules/admin/verbs/possess.dm]: what kind of shitty code cares about indentation

**Review by Amelia0010 — APPROVED** [review]

**Comment by ThatLing** [comment]
ThatLing/(soap): Nice

File: code/modules/admin/verbs/possess.dm [hunk 2]
```diff
@@ -25,11 +25,17 @@
 	usr.control_object = O
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Possess Object") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-/proc/release()
+/proc/release(obj/O in world) //yogs
```
Comment (ThatLing) [inline code/modules/admin/verbs/possess.dm]:
```suggestion
/proc/release(obj/O in world) // yogs - fixed release object
```

File: code/modules/admin/verbs/possess.dm [hunk 3]
```diff
@@ -25,11 +25,17 @@
 	usr.control_object = O
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Possess Object") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-/proc/release()
+/proc/release(obj/O in world) //yogs
 	set name = "Release Obj"
 	set category = "Object"
 	//usr.loc = get_turf(usr)
 
+//YOGS start
```
Comment (ThatLing) [inline code/modules/admin/verbs/possess.dm]:
```suggestion
// yogs start - fixed release object
```

File: code/modules/admin/verbs/possess.dm [hunk 4]
```diff
@@ -25,11 +25,17 @@
...
+/proc/release(obj/O in world) //yogs
 	set name = "Release Obj"
 	set category = "Object"
 	//usr.loc = get_turf(usr)
 
+//YOGS start
+	if(!usr.control_object)
+		to_chat(usr, "You need to possess an object first!")
+		return
+//YOGS end
```
Comment (ThatLing) [inline code/modules/admin/verbs/possess.dm]:
```suggestion
// yogs end
```

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/possess.dm (hunk 4 above)
Comment (Moja10) [reply code/modules/admin/verbs/possess.dm]: Is the comment required to be case sensitive or not? Asking for a friend

Comment (Ktlwjec) [reply code/modules/admin/verbs/possess.dm]: also asking for a friend

**Pull request merged**